core/libs/input/input.php

```diff
@@ -180,20 +180,21 @@ public static function filter($var)
      * @param string $str clave a usar
      * @return mixed
      */
-    protected static function getFilter(Array $var, $str){
-        if(empty($str))
+    protected static function getFilter(Array $var, $str)
```

Comment:
__[PHP_CodeSniffer]__

PHP keywords must be lowercase; expected "array" but found "Array"


<!-- Comment by [SideCI](https://sideci.com) -->